src/scheduler/workers/unpublish/weekly/types.ts

```diff
@@ -0,0 +1,20 @@
+import { Address, List, ListItem } from "@prisma/client";
+
+export interface Meta {
+  /**
+   * annual review reference for list (`List.jsonData.currentAnnualReview.reference`)
+   */
+  reference: string;
+
+  /**
+   * Weeks until unpublish. Used for debugging. added to `Event.jsonData.notes`.
+   */
+  weeksUntilUnpublish: number;
+
+  parsedUnpublishDate: string;
+
+  countryName: string;
+}
+
+export type ListItemWithCountryName = ListItem & { address: Address & { country: { name: string } } };
+export type ListWithCountryName = List & { country: { name: string } };
```

Comment:
Would it make sense to have a new interface for `Country`?

```suggestion
interface Country {
 country: {
   name: string
};
export type ListItemWithCountryName = ListItem & { address: Address & Country };
export type ListWithCountryName = List & Country;
```

Reply:
a type would make more sense (since `Prisma` primarily returns types rather than interfaces), but I get your point, just to tidy up/alias the types.

`Country` already exists in prisma, but it includes all of the column/fields (id, name, updated, created etc). When you do 

```
include: {
  country: {
   select: {
    name: true 
   }
 }
}
```

the prisma type system (uses a lot of generics and conditional types), will return me a type that looks like this:
![image](https://user-images.githubusercontent.com/22080510/222427416-c91978ad-fd0f-4047-bd65-c701a536d54b.png)

anyway, yeah, I'll create a new type for it anyway to tidy it up.